src/main/kotlin/net/ccbluex/liquidbounce/features/misc/HideAppearance.kt

```diff
@@ -38,20 +40,55 @@ import org.lwjgl.glfw.GLFW
 import java.lang.Thread.sleep
 import kotlin.concurrent.thread
 
+private val modMenuPresent = runCatching {
+    Class.forName("com.terraformersmc.modmenu.ModMenu")
+    true
+}.getOrDefault(false)
+
 /**
  * Hides client appearance
  *
  * using 2x CRTL + SHIFT to hide and unhide the client
  */
 object HideAppearance : EventListener {
 
+    /**
+     * These mods will be removed from ModMenu.
+     * When [isHidingNow] is true
+     * Or added, if [isHidingNow] is false
+     *
+     * Because we don't know about the [Mod] container on each mod in this list
+     * We set the default value is null.
+     * And we'll provide the value after first removing the mod
+     */
+    private val modContainersToHide: MutableMap<String, Mod?> = arrayOf(
+        "liquidbounce", "mcef"
+    ).associateWith { null }.toMutableMap()
+
     private val shiftChronometer = Chronometer()
 
     var isHidingNow = false
         set(value) {
             field = value
             RenderSystem.recordRenderCall(::updateClient)
+
+            if (modMenuPresent) {
+                if (value) {
+                    for (id in modContainersToHide.keys) {
+                        modContainersToHide[id] = ModMenu.MODS.remove(id)
+                        ModMenu.ROOT_MODS.remove(id)
```

Comment:
Create a .java for accessing Mod Menu as we do with ViaFabricPlus.
https://github.com/CCBlueX/LiquidBounce/blob/nextgen/src/main/java/net/ccbluex/liquidbounce/utils/client/vfp/VfpCompatibility.java

The reason for this is that the Kotlin compiler sometimes tries to access the class before it's supposed to be used, causing a ClassNotFound if the mod is not installed. It may not happen in this case - but I'd rather be sure it doesn't.